include/JSystem/JUtility/JUTProcBar.h

```diff
@@ -1,6 +1,91 @@
 #ifndef JUTPROCBAR_H
 #define JUTPROCBAR_H
 
+#include "JSystem/JKernel/JKRHeap.h"
+#include "JSystem/JUtility/TColor.h"
 #include "dolphin/types.h"
 
+class JUTProcBar {
+public:
+    class CTime {
+    public:
+        /* 802E7340 */ CTime();
+
+        void start(u8 param_0, u8 param_1, u8 param_2) {
+            field_0x10 = param_0;
+            field_0x11 = param_1;
+            field_0x12 = param_2;
+            mTick = OSGetTick();
+        }
+
+        void end() {
+            field_0x4 = ((OSGetTick() - mTick) * 8) / ((*(u32*)0x800000F8 >> 2) / 125000);
```

Comment:
```suggestion
            field_0x4 = ((OSGetTick() - mTick) * 8) / ((*(u32*)0x800000F8 / 4) / 125000);
```
Does this work? I think it makes more sense to write everything as multiplication/division